package.json

```diff
@@ -47,9 +60,11 @@
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.17.3",
-    "eslint-plugin-mocha": "^5.3.0",
-    "mocha": "^6.1.4",
```

Comment:
Good job bro. Please, could you explain why mocha was removed from the dev dependencies?

Reply:
I was trying to resolve a merge confilct, thank you for pointing it out, i'll add it back